.changeset/great-kangaroos-confess.md

```diff
@@ -0,0 +1,19 @@
+---
+'xstate': major
```

Comment:
```suggestion
'xstate': patch
```

Reply:
Wouldn't this technically be `minor`?

Also I've been wondering about how this works with the `next` branch. When I ran `changesets add` for #3997, it just assumed it was a `major` change for some reason.

Do the different versions have any effect when it's in prerelease mode?

Reply:
> Wouldn't this technically be minor?

That works too. I suggest `patch` here since this was an oversight on our part. This kinda should be supported from the start. I don't mind using minor though - we'll have to clean up this changeset before releasing final v5 anyway (this particular changeset is only relevant for the prerelease)

> When I ran changesets add for https://github.com/statelyai/xstate/pull/3997, it just assumed it was a major change for some reason.

That's weird, it should allow u to select the type.

> Do the different versions have any effect when it's in prerelease mode?

It's just for grouping them together. And all pending changesets are used to determine the next version - if we didn't have any major changesets then we would end up with `xstate@4.38.0-beta.11` (or similar).

Reply:
Thanks for the explanation! Makes sense 👍